music_assistant/providers/spotify/streaming.py

```diff
@@ -28,18 +29,33 @@ async def get_audio_stream(
         self, streamdetails: StreamDetails, seek_position: int = 0
     ) -> AsyncGenerator[bytes, None]:
         """Return the audio stream for the provider item."""
-        spotify_uri = f"spotify://track:{streamdetails.item_id}"
+        # Ensure librespot binary is available
+        if not self.provider._librespot_bin:
+            raise AudioError(f"Librespot binary not available for {streamdetails.item_id}")
+
+        # Use explicit media type from streamdetails
+        if hasattr(streamdetails, "media_type") and streamdetails.media_type:
+            is_episode = streamdetails.media_type == MediaType.PODCAST_EPISODE
+        else:
+            # Fallback to bitrate detection if media_type not available
+            is_episode = streamdetails.audio_format.bit_rate == 160
+
+        media_type = "episode" if is_episode else "track"
+        bitrate = "160" if is_episode else "320"
```

Comment:
why is this needed ? I'm assuming that librespot is smart enough to pick a lower bitrate if the requested one is not available